core/src/main/java/org/elasticsearch/action/admin/cluster/allocation/ClusterAllocationExplanation.java

```diff
@@ -119,6 +119,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             builder.field("index", shardRouting.getIndexName());
             builder.field("shard", shardRouting.getId());
             builder.field("primary", shardRouting.primary());
+            String shardState = shardRouting.state().toString().toLowerCase(Locale.ROOT);
```

Comment:
this was a mistake, i meant to use it in the line below as well, so I changed the line below to use it (its used twice now)